test/state/precompiles.cpp

```diff
@@ -443,20 +443,6 @@ ExecutionResult bls12_g1add_execute(const uint8_t* input, size_t input_size, uin
     return {EVMC_SUCCESS, 128};
 }
 
-ExecutionResult bls12_g1mul_execute(const uint8_t* input, size_t input_size, uint8_t* output,
```

Comment:
Remove also their `analysis` counterparts for g1 and g2. Here and from the header.

Reply:
Removed.